firefox-ios/Client/Frontend/Library/Bookmarks/BookmarksViewController.swift

```diff
@@ -393,6 +393,12 @@ class BookmarksViewController: SiteTableViewController,
         presentContextMenu(for: indexPath)
     }
 
+    @objc
+    private func didPressContextMenuButton(_ sender: UIButton) {
+        let row = sender.tag
+        let indexPath = IndexPath(row: row, section: 0) // Assuming section 0, adjust if needed
+        presentContextMenu(for: indexPath)    }
+
```

Comment:
indentation it seems here
```suggestion
        presentContextMenu(for: indexPath)    
    }

```